<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QFrame" name="frame" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QHBoxLayout" name="mp_opponentContainer" />
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>List of player widgets - later this component will be nicer.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>258</width>
        <height>572</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QFrame" name="mp_chatFrame" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet" >
         <string notr="true" >background-color: rgba(0, 0, 0, 15);</string>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="mp_playerFrame" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet" >
         <string notr="true" >background-color: rgba(0, 0, 0, 15);</string>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="mp_logFrame" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet" >
         <string notr="true" >background-color: rgba(0, 0, 0, 15);</string>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>26</height>
    </rect>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuGame" >
    <property name="title" >
     <string>&amp;Game</string>
    </property>
    <addaction name="mp_actionConnectToServer" />
    <addaction name="mp_actionDisconnectFromServer" />
    <addaction name="separator" />
    <addaction name="mp_actionJoinGame" />
    <addaction name="mp_actionLeaveGame" />
    <addaction name="mp_actionExit" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure_Profile" />
   </widget>
   <addaction name="menuGame" />
   <addaction name="menuTools" />
  </widget>
  <widget class="QStatusBar" name="mp_statusbar" />
  <action name="mp_actionConnectToServer" >
   <property name="icon" >
    <iconset>
     <normaloff>:/icons/images/icon-connect.svg</normaloff>:/icons/images/icon-connect.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Connect to Server</string>
   </property>
  </action>
  <action name="mp_actionExit" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="mp_actionDisconnectFromServer" >
   <property name="text" >
    <string>&amp;Disconnect from Server</string>
   </property>
  </action>
  <action name="mp_actionJoinGame" >
   <property name="text" >
    <string>&amp;Join Game</string>
   </property>
  </action>
  <action name="mp_actionLeaveGame" >
   <property name="text" >
    <string>&amp;Leave Game</string>
   </property>
  </action>
  <action name="actionConfigure_Profile" >
   <property name="text" >
    <string>Configure Profile</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="client.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mp_actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
